llvm/test/MC/AArch64/SVE2/adclb.s

```diff
@@ -2,6 +2,8 @@
 // RUN:        | FileCheck %s --check-prefixes=CHECK-ENCODING,CHECK-INST
 // RUN: llvm-mc -triple=aarch64 -show-encoding -mattr=+sme < %s \
 // RUN:        | FileCheck %s --check-prefixes=CHECK-ENCODING,CHECK-INST
+// RUN: llvm-mc -triple=aarch64 -show-encoding -mattr=+sme-fa64 < %s \
```

Comment:
There isn't much value in (updating all) these tests. The main point of `sme-fa64` is for it to be used in conjunction with SME, to tell that full instruction set is available in streaming SVE mode.

Reply:
I've removed them all (some whitespace in some survived though somehow :facepalm:)